example/main.go

```diff
@@ -27,15 +28,19 @@ func main() {
 		iscsi.EnableDebugLogging(os.Stdout)
 	}
 
+	targets, err := parseTargets(tgtp, *iqn)
```

Comment:
Would it be worth adding a check here for a empty targets?

Reply:
Good point :pray:. Done :+1:.

Reply:
@j-griffith any more feedback? Is this mergeable? Thanks.

Reply:
@j-griffith gentle ping :pray:.